user-guide/Reference/Metrics/swarming_elements_benchmarks.md

```diff
@@ -0,0 +1,27 @@
+---
+uid: swarming_bookings_benchmarks
```

Comment:
bookings

Reply:
Guess where I copy and pasted from to get started :)